sdk/generic-sdk-core/generic-core-http-okhttp/src/main/java/com/generic/core/http/okhttp/implementation/OkHttpResponse.java

```diff
@@ -11,13 +11,15 @@
 /**
  * Default HTTP response for OkHttp.
  */
-public final class OkHttpResponse extends OkHttpResponseBase {
+public final class OkHttpResponse extends OkHttpResponseBase<byte[]> {
```

Comment:
With our usage of `BinaryData` everywhere I think `OkHttpBufferedResponse` and `OkHttpResponse` can be merged and the appropriate buffering handling and `getValue` can be determined by `BinaryData`'s APIs